api/helpers/errors.py

```diff
@@ -0,0 +1,33 @@
+# 400 - Server cannot process request
+# 401 - Authentication required or has failed
+# 403 - Authorization level not met
+# 404 - Not Found
+# 405 - Incorrect HTTP Verb
+# 406 - Input data is incorrectly formatted
+# 410 - Empty resource
+# 200 - Standard sucess
+# 201 - Creates new resource
+# 202 - Accepted information for processing (update, destroy)
+# 204 - No Content (maybe this is delete)
```

Comment:
I'm happy this exists.  It's very helpful and informative.

Reply:
Yay! Feel free to add any others we may use